player/lua/stats.lua

```diff
@@ -1721,20 +1721,20 @@ mp.add_key_binding(nil, "display-stats", function() process_key_binding(true) en
 mp.add_key_binding(nil, "display-stats-toggle", function() process_key_binding(false) end,
     {repeatable=false})
 
-for k, _ in pairs(pages) do
+for _, page in pairs(pages) do
     -- Single invocation key bindings for specific pages, e.g.:
     -- "e script-binding stats/display-page-2"
-    mp.add_key_binding(nil, "display-page-" .. k, function()
-        curr_page = k
+    mp.add_key_binding(nil, "display-page-" .. page.idx, function()
+        curr_page = page.idx
```

Comment:
This should still use a key for cur_page value
```suggestion
        curr_page = k
```

Reply:
Thanks, I was getting a little agitated.